### PR TITLE
[MAINT] add initial packaging requirements

### PR DESCRIPTION
**Pull request by jerdra**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+long_description = file:README.md
+keywords = neuroimaging, etl, automation, orchestration
+classifiers =
+	Intended Audience :: Science/Research
+	Topic :: Scientific/Engineering
+	Programming Language :: Python :: 3.7
+	Programming Language :: Python :: 3.8
+	Programming Language :: Python :: 3.9
+project_urls =
+	GitHub = https://github.com/tigrlab/airtigrs
```
Comment (josephmje) [inline setup.cfg:13]: Should the repository name be changed to `airtigrs` or the text here to `air-tigrs`?

```suggestion
	GitHub = https://github.com/tigrlab/airtigrs
```

**Review by josephmje — COMMENTED** [review]
Looks good to me!

One additional question I had was whether the `dags` folder should be excluded from the package install?

**Review by slimnsour — APPROVED** [review]
Each build target worked for me, looks good!

File: setup.cfg (hunk 1 above)
Comment (jerdra) [reply setup.cfg:13]: Yupp good catch!

**Pull request merged**